source/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,58 @@
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+You can delete a single document from a collection using the ``deleteOne()`` 
+method on a ``MongoCollection`` object. The method accepts a query filter
+that matches the document you want to delete. If you do not specify
+a filter, MongoDB matches the first document in the collection. The 
+``deleteOne()`` method only deletes the first document matched.
+
+This method returns an instance of ``DeleteResult`` which contains information 
+including how many documents were deleted as a result of the operation.
+
+If your delete operation fails, the driver raises an exception. For more
+information on the types of exceptions raised under specific conditions,
+see the API documentation for ``deleteOne()``, linked at the bottom of
+this page.
```

Comment:
Why not just use inline links to `deleteOne()` and `DeleteResult`? Feels a little annoying to have to scroll down to the bottom.

Reply:
This is in response to some feedback from drivers teams that the inline links pulled them away from the content of the page too early. I can revisit this again at some point if it seems like this is makes it difficult for the docs to use.